client/my-sites/checkout/checkout/index.jsx

```diff
@@ -38,7 +38,11 @@ import {
 	hasTransferProduct,
 	jetpackProductItem,
 } from 'lib/cart-values/cart-items';
-import { JETPACK_BACKUP_PRODUCTS, JETPACK_SEARCH_PRODUCTS } from 'lib/products-values/constants';
+import {
+	JETPACK_BACKUP_PRODUCTS,
```

Comment:
Instead of always having to import a new product constant we can use `isJetpackProductSlug`